src/AppDataSource.ts

```diff
@@ -0,0 +1,23 @@
+import { DataSource, DataSourceOptions } from "typeorm";
+import dotenv from "dotenv";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+dotenv.config();
```

Comment:
Make sure that it's necessary doing this dotenv config once again (it already was done in index.ts)